.github/workflows/stacks-core-tests.yml

```diff
@@ -135,25 +138,45 @@ jobs:
   # Core contract tests
   core-contracts-clarinet-test:
     name: Core Contracts Test
+    defaults:
+      run:
+        working-directory: "./contrib/core-contract-tests/"
     runs-on: ubuntu-latest
     steps:
-      ## Checkout the code
       - name: Checkout the latest code
         id: git_checkout
-        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-
-      ## Use Clarinet to run contract unit-tests and create code coverage file
-      - name: Execute core contract unit tests in Clarinet
+        uses: actions/checkout@v3
+      - name: Execute core contract unit tests with clarinet-sdk
         id: clarinet_unit_test
-        uses: docker://hirosystems/clarinet:1.8.0
+        uses: actions/setup-node@v3
         with:
-          args: test --coverage --manifest-path=./contrib/core-contract-tests/Clarinet.toml
-
+          node-version: 18.x
+          cache: "npm"
+          cache-dependency-path: "./contrib/core-contract-tests/package-lock.json"
+      - run: npm ci
+      - run: npm test
       ## Upload code coverage file
       - name: Code Coverage
         id: codecov
         uses: stacks-network/actions/codecov@main
         with:
           test-name: ${{ matrix.test-name }}
           upload-only: true
-          filename: ./coverage.lcov
+          filename: ./lcov.info
+
+  # Core contract tests on Clarinet v1
+  # Check for false positives/negatives
+  # https://github.com/stacks-network/stacks-blockchain/pull/4031#pullrequestreview-1713341208
+  core-contracts-clarinet-test-clarinet-v1:
```

Comment:
Hey! How is this different than `core-contract-clarinet-test-clarinet` above?

Reply:
When i did the merge from master->develop last week, this was something that was removed that @hugocaillard had worked on. 
the clarinet docker image is something we found will no longer work with the `latest` image, so work was done to use the clarinet sdk vs the docker image. 

i envision this `v1` test can be removed in time, but i felt it worthwhile to keep since it was there before the merge from master.

Reply:
More context in the link in the comment above 👌 
https://github.com/stacks-network/stacks-core/pull/4031#pullrequestreview-1713341208